packages/ui/src/themes/screens.styled.ts

```diff
@@ -3,6 +3,8 @@ export const screens = {
   md: '1024px',
   lg: '1440px',
   xl: '2560px',
+
+  hlg: '950px',
```

Comment:
What does `hlg` stand for? Looks like it needs to be `sm`

Reply:
hlg means height large and we can't rename it to sm as it's large for heights